Add support for encoding/decoding u128 values

### Description of feature:

stdint.rs currently declares support for uints up to 64. u128 could be easily supported but its currently not. External clients cannot manually add this support due to:

> error[E0117]: only traits defined in the current crate can be implemented for primitive types 

### Use case:

u128 are typically used for encoding uuids. In my case, I specifically need it as iOS's IDFV is a uuid. Adding support seems pretty much free.

### Implementation discussion
Change is a one liner in stdint.rs to add `impl_stdint!(u128);`

--- a/mls-rs-codec/src/stdint.rs
+++ b/mls-rs-codec/src/stdint.rs
@@ -32,6 +32,7 @@ impl_stdint!(u8);
 impl_stdint!(u16);
 impl_stdint!(u32);
 impl_stdint!(u64);
+impl_stdint!(u128);
 
 #[cfg(test)]
 mod tests {
@@ -80,5 +81,18 @@ mod tests {
         let recovered = u64::mls_decode(&mut &*serialized).unwrap();
 
         assert_eq!(recovered, 100000000000u64);
+    }
+
+    #[test]
+    fn u128_round_trip() {
+        let serialized = 10000000000000000u128.mls_encode_to_vec().unwrap();
+        assert_eq!(
+            serialized,
+            vec![0, 0, 0, 0, 0, 0, 0, 0, 0, 35, 134, 242, 111, 193, 0, 0]
+        );
+
+        let recovered = u128::mls_decode(&mut &*serialized).unwrap();
+
+        assert_eq!(recovered, 10000000000000000u128);
     }
 }

